README.md

```diff
@@ -39,22 +38,15 @@ data is stored in local storage. This data is retrieved from local storage anyti
 
 <details>
   <summary>Server</summary>
-  <ul>
-    <li><a href="#"></a></li>
-  </ul>
 </details>
 
 <details>
 <summary>Database</summary>
-  <ul>
-    <li><a href="#"></a></li>
-  </ul>
 </details>
 
```

Comment:
- [ ] Kindly remove these sections Server and Database since you don't have any backend and database :thinking: Kindly remove both of these to make your readme descriptive and professional :blush: